src/main/java/org/spongepowered/common/data/processor/data/DyeableDataProcessor.java

```diff
@@ -59,11 +59,16 @@ public DyeableDataProcessor() {
     }
 
     public static boolean isDyeable(Item item) {
-        return item.equals(Items.dye)
-                || Block.getBlockFromItem(item).equals(Blocks.wool)
-                || Block.getBlockFromItem(item).equals(Blocks.stained_glass)
-                || Block.getBlockFromItem(item).equals(Blocks.stained_glass_pane)
-                || Block.getBlockFromItem(item).equals(Blocks.stained_hardened_clay);
+        if (item.equals(Items.dye)) {
+            return true;
+        }
+
+        Block bl = Block.getBlockFromItem(item);
```

Comment:
Old habits die hard! In the context of this, I can't see it mattering much (it should be obvious from the context), but that's all personal style I guess.

Sorted it.